subsys/fs/nvs/nvs.c

```diff
@@ -425,12 +456,14 @@ static int nvs_sector_close(struct nvs_fs *fs)
 	int rc;
 	struct nvs_ate close_ate;
 	size_t ate_size;
+	uint16_t sector_offset;
 
 	ate_size = nvs_al_size(fs, sizeof(struct nvs_ate));
+	sector_offset = (uint16_t)((fs->ate_wra	+ ate_size) & ADDR_OFFS_MASK);
 
 	close_ate.id = 0xFFFF;
 	close_ate.len = 0U;
-	close_ate.offset = (uint16_t)((fs->ate_wra + ate_size) & ADDR_OFFS_MASK);
+	close_ate.offset = sector_offset;
```

Comment:
why this `sector_offset` assignment mediation was introduced?

Reply:
Because it was over the 80 char limit.

Reply:
line should be break instead (I think).